docs/ReferenceGuides/UnderscoredAttributes.md

```diff
@@ -117,6 +117,27 @@ extension Text {
 }
 ```
 
+## `@_documentation(metadata: ...)`
```

Comment:
I assume the value of the `metadata` parameter is just a string that be specified with or without quotes?

Reply:
Right now it can only be an identifier, but i could see about making it a quoted string as well.

Reply:
If it's a string in quotes, developers could stick in useful metadata like JSON, CSV, or hex encoded images which would be fun 👌🏼